Code/CodeIndex.md

```diff
@@ -1,8 +1,9 @@
 # Code Index
-A list of the code contributed by our community members. This is either hosted here on the CMAS open-code repository, or externally. 
+A list of the code contributed by our community members. This is either hosted here on the CMAS open-code repository, or externally.
 When you add a new entry, make sure you include a link to the code, the required software, and your name, so we know who to thank! :heart::clap:
 
-- [Gait Profile Score & Movement Analysis Profile](https://github.com/cmasuki/open-code/tree/master/Code/Gait_profile_score) by [Neil Postans](https://github.com/npostans). 
+- [Gait Profile Score & Movement Analysis Profile](https://github.com/cmasuki/open-code/tree/master/Code/Gait_profile_score) by [Neil Postans](https://github.com/npostans).
 Needs Python 2.7 (32 bit version) & Vicon Nexus 2.x
 - [Local Dynamic Stability code (Rosensteins Algorithm)](https://github.com/SjoerdBruijn/LocalDynamicStability) by [Sjoerd Bruijn](https://github.com/SjoerdBruijn) Needs Matlab (any version)
 - [Code to read accelerometry data from a mobile phone via Matlab in realtime](https://github.com/SjoerdBruijn/StreamPhoneData) by [Sjoerd Bruijn](https://github.com/SjoerdBruijn) Needs Matlab (any version)
+- [visualisation and statistical processing of 3D gait data with R](https://github.com/pyCGM2/rCGM2) bu [Fabien Leboeuf](https://github.com/pyCGM2/rCGM2) Needs Rstudio
```

Comment:
Your name should link to your github profile. (The name of the package should link to the package itself)